### DISCUSSION TITLE
Problem compiling tdslite 5.0

### DISCUSSION THREAD

**Discussion by bradbeshaw** [tdslite/tdslite#48] [Q&A]
The error following occurs when attempting to compile my sketch or any of the examples. Where can I find this file?
 
c:\Projects\Arduino\libraries\tdslite\src\tdslite-net\asio\src\tdsl_netimpl_asio.cxx:24:10: fatal error: boost/asio.hpp: No such file or directory
   24 | #include <boost/asio.hpp>

Thank you for any assistance.

**Comment by xmkg** [comment]
Hello @bradbeshaw 

How are you trying to compile your example? I don't see such an error at all. Please use the official Arduino and Platform.IO releases via their respective library manager(s).

**Comment by bradbeshaw** [comment]
I uninstalled everything. IDE, libraries, sketches. Re-installed IDE, boards, and tdslite.
Same result. Might be related to my ESP32S3 board libraries.

**Reply by xmkg** [reply]
I have a similar board (NodeMCU ESP32s), let me check again. Which IDE are you using by the way? Arduino IDE v1/v2 or something else?

**Reply by xmkg** [reply]
I've tried compiling the [esp.cpp](https://github.com/tdslite/tdslite/blob/main/tests/sketches/esp/esp.cpp) sketch for a few different ESP32 boards and they all compiled fine. This has been done using Arduino IDE 2.2.1 on Linux and tdslite 0.5.0. Can you try compiling the said sketch in your environment and let me know how that goes?

**Reply by xmkg** [reply]
Hi @bradbeshaw 

Have you been able to solve the issue?

**Comment by xmkg** [comment]
Hi @bradbeshaw 

FYI, this issue should be fixed with the 0.8.0 release.
